packages/next-app/src/polyfills.ts

```diff
@@ -0,0 +1,7 @@
+import { Buffer } from "buffer";
```

Comment:
if this is also related to wagmi can you leave a comment in the top of the file explaining this?

Reply:
yup!